packages/ui/src/middleware/admin.js

```diff
@@ -0,0 +1,22 @@
+/* global window */
+import {
+  disableAdminMode,
+  enableAdminMode
+} from '../state/ducks/admin'
+
+export default ({ dispatch, getState }) => next => action => {
+  // This is a **hack** to expose admin functionality through the console
+  // TODO: Unhack it! Make a button or something.
+  if (typeof window && !('__HACKS__' in window)) {
+    window.__HACKS__ = () => {
```

Comment:
A little gross, but firing `__HACKS__()` in the console reveals some Admin links.